cmd/cosign/cli/verify.go

```diff
@@ -29,89 +29,107 @@ import (
 	"github.com/sigstore/cosign/pkg/cosign/fulcio"
 )
 
+// VerifyCommand verifies a signature on a supplied container image
+type VerifyCommand struct {
+	KmsVal      string
+	Key         string
+	CheckClaims bool
+	Annotations map[string]string
+}
+
+// Verify builds and returns an ffcli command
 func Verify() *ffcli.Command {
-	var (
-		flagset     = flag.NewFlagSet("cosign verify", flag.ExitOnError)
-		kmsVal      = flagset.String("kms", "", "verify via a public key stored in a KMS")
-		key         = flagset.String("key", "", "path to the public key")
-		checkClaims = flagset.Bool("check-claims", true, "whether to check the claims found")
-		annotations = annotationsMap{}
-	)
+	cmd := VerifyCommand{}
+	flagset := flag.NewFlagSet("cosign verify", flag.ExitOnError)
+	annotations := annotationsMap{}
+
+	flagset.StringVar(&cmd.Key, "kms", "", "verify via a public key stored in a KMS")
```

Comment:
nit: i think the help text on this one is incorrect/copy-pasted.

Reply:
Oops - great pickup, thanks @dlorenc. Fixed now.